### PR TITLE
Removed unused imports

### PR DESCRIPTION
**Pull request by banjaxbanjo**
Removing left over imports that are currently not used and grayed out.

### CODE REVIEW

**Comment by SecUpwN** [comment]
Thanks for this pull request, @banjaxbanjo! Short reminder: Please increase the version as described [here](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/wiki/Development-Status#versioning) after you fixed the failing Travis CI build due to the following error:

```
/home/travis/build/SecUpwN/Android-IMSI-Catcher-Detector/app/src/main/java/com/SecUpwN/AIMSICD/utils/MiscUtils.java:194: error: cannot find symbol

Calendar calendar = Calendar.getInstance();

^

symbol: class Calendar

location: class MiscUtils

/home/travis/build/SecUpwN/Android-IMSI-Catcher-Detector/app/src/main/java/com/SecUpwN/AIMSICD/utils/MiscUtils.java:194: error: cannot find symbol

Calendar calendar = Calendar.getInstance();
```

**Comment by banjaxbanjo** [comment]
@SecUpwN  Ah sorry didnt mean to merge the other code with the first PR of removing the unused imports. Was missing a import for Calendar.

**Comment by SecUpwN** [comment]
>Ah sorry didnt mean to merge the other code with the first PR of removing the unused imports.

Don't worry, you#re doing awesome job. @agilob, would you please check this PR to avoid "Issues"? ;-)

**Comment by banjaxbanjo** [comment]
@SecUpwN Will I close this and make a fresh PR or is this going to merge now since I fixed the import error?

**Comment by SecUpwN** [comment]
>Will I close this and make a fresh PR or is this going to merge now since I fixed the import error?

Do you see the green button? It is now perfectly mergeable, I'm just waiting for @agilob to check this pull request since I'd like to give him the chance to see your code after the discussion before. Chill. ;-)

File: app/src/main/java/com/SecUpwN/AIMSICD/smsdetection/SmsDetector.java [hunk 1]
```diff
@@ -181,53 +168,24 @@ public void checkForSilentSms(String[] progress){
 
         for(int arrayindex = 0;arrayindex < SILENT_ONLY_TAGS.length;arrayindex++)
         {
-            int MAX_INT = progress.length -4;
+
             for(int x =0;x <progress.length;x++) {//check all progress buffer for strings
 
                 if(progress[x].length() < 250){//check only short strings for faster processing
-                     /*
```
Comment (agilob) [inline app/src/main/java/com/SecUpwN/AIMSICD/smsdetection/SmsDetector.java]: and... now it's safe to remove that code -.-'

**Comment by banjaxbanjo** [comment]
Trying to keep everyone happy here so I removed the big block that was commented out to save headache but will put some of it back in soon as the -v time is now a good method for logging data with an accurate timestamp.

**Comment by SecUpwN** [comment]
@agilob, are you happy with the PR now? Give me a green light for merging, please.

**Comment by agilob** [comment]
sure XD

**Comment by banjaxbanjo** [comment]
;-)

**Comment by agilob** [comment]
When will it be in fdroid?

**Comment by SecUpwN** [comment]
Thank you! That's what I call "live development", guys. ;-)

**Pull request merged**